LICENSE

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
In fact we're Apache 2 everywhere you're right. Can you just update the commit title?